cmd/machine-api-migration/main.go

```diff
@@ -153,6 +166,32 @@ func main() {
 		os.Exit(0)
 	}
 
+	k8sclient := mgr.GetClient()
+	infra := &configv1.Infrastructure{}
+
+	err = k8sclient.Get(stop, client.ObjectKey{Name: controllers.InfrastructureResourceName}, infra)
+	if err != nil {
+		klog.Errorf("failed to fetch infrastructure: %s", err)
+		os.Exit(1)
+	}
```

Comment:
Nit, inline this and scope the error variable to just the if statement
```suggestion
	if err := k8sclient.Get(stop, client.ObjectKey{Name: controllers.InfrastructureResourceName}, infra); err != nil {
		klog.Errorf("failed to fetch infrastructure: %s", err)
		os.Exit(1)
	}
```